### DISCUSSION TITLE
How to Google sign via FlutterFire UI (clientId?)

### DISCUSSION THREAD

**Discussion by yasseralsaidi** [firebase/flutterfire#7653] [Q&A]
i try to use the new FlutterFire UI (exactly for Auth)

source: https://firebase.flutter.dev/docs/ui/auth/integrating-your-first-screen

 i want to find the (clientId) in Firebase Console.

**Comment by bclarksoftware** [comment]
I'm struggling with this as well while reading the description within the Facebook provider section https://firebase.flutter.dev/docs/ui/auth/configuring-providers#facebook.

It mentions "the configuration requires the clientId property (which can be found in the Firebase Console) to be set for seamless cross-platform support." Where exactly is the clientId? When generating the FirebaseOptions.dart file, the FirebaseOptions for android does not specify the client id. Do we use the appId instead?

In my case, I'm able to get to the point of clicking "Continue" with Facebook, but it just doesn't create the user within the Authentication section of Firebase.

Update: I got mine working by using one of the Facebook testing users rather than my own Facebook account. I'm assuming I need to be in "Live" mode in order to use my own account for testing purposes. As for the clientid, it doesn't appear to matter what I put as a string for that variable. Will this be a problem for me in the future?
